app/src/main/java/atlant/moviesapp/activity/EpisodeActivity.java

```diff
@@ -50,16 +55,17 @@
 
     @BindView(R.id.cast_recycler_view)
     RecyclerView castRecyclerView;
-
+    private StringUtils stringUtils;
```

Comment:
There is no need for this, wherever you need to call a method from a Utils class. Just call its static method from the class. No need for a field of that type at all.